kubectl-plugin/waiops-status.md

```diff
@@ -426,6 +432,7 @@ git clone https://github.com/IBM/cp4waiops-samples.git
 chmod +x cp4waiops-samples/kubectl-plugin/kubectl-waiops
 cp cp4waiops-samples/kubectl-plugin/kubectl-waiops /usr/local/bin/kubectl-waiops
 
+oc waiops multizone-status
```

Comment:
Hey Adam, I see there's this command and `oc waiops multizone status` at L416. This one doesn't work.

```
[root@api.pours.cp.fyre.ibm.com cp4waiops-samples]# oc waiops multizone-status
error: unknown command "multizone-status"
See 'oc waiops -h' for help and examples
```

Reply:
Thanks @lsalazar1 done.

Reply:
Thanks!